web/src/domain/employees.ts

```diff
@@ -28,3 +28,8 @@ export interface EmployeesFilters {
 	order: SortingDirection;
 	role?: Employee["role"];
 }
+
+/**
+ * Sortable fields of a employees.
+ */
+export type EmployeeRoles = Employee["role"];
```

Comment:
```suggestion
export type EmployeeRole = Employee["role"];
```
This should in singular since `Employee["role"]` is a union